OrbitGl/CaptureWindow.cpp

```diff
@@ -519,9 +519,11 @@ void CaptureWindow::Draw() {
     time_graph_.GetBatcher().DrawLayer(layer, GetPickingMode() != PickingMode::kNone);
     ui_batcher_.DrawLayer(layer, GetPickingMode() != PickingMode::kNone);
 
-    PrepareScreenSpaceViewport();
-    text_renderer_.DisplayLayer(&ui_batcher_, layer);
-    RenderText(layer);
+    if (GetPickingMode() == PickingMode::kNone) {
+      PrepareScreenSpaceViewport();
```

Comment:
This method couldn't be in that if, or we have to add it again before also. Probably we can implement it in a different way (like having a bool flag related on which viewport we are), but we need to switch to the screen space viewport not only for the text, also because on some z-layer elements (the ones who are in the screen-space z-layer).

Reply:
Right, thanks, I moved it out.